File: 지은/Lv.2/단속카메라.py
def solution(routes):
    routes.sort(key=lambda x: x[1])
    count = 0

    curr_last = -float('Inf')
    for route in routes:
        if curr_last >= route[0]:
            continue
        else:
            curr_last = route[1]
            count += 1
    return count

File: 화연/Lv.3/단속카메라.js
/**
 * https://school.programmers.co.kr/learn/courses/30/lessons/42884
 * @param {number[][]} routes 
 * @returns 
 */
function solution(routes) {
  let answer = 0;
  const outRoutes = [...routes].sort(([, a], [, b]) => a - b);
  let outLine = 0;
  for(let i = 0; i < outRoutes.length; i++) {
    const [inPoint, outPoint] = outRoutes[i];
    if(i === 0) {
      outLine = outPoint;
      answer++;
      continue;
    }
    if(inPoint > outLine) {
      outLine = outPoint;
      answer++;
    }
  }
  return answer;
}